ons_alpha/jinja2/templates/base_page.html

```diff
@@ -34,23 +35,30 @@
 
 {% set languages=translation_urls() %}
 
-{#{% block header %}#}
-{#    {% secondary_nav %}#}
-{#    {% primary_nav %}#}
-{#{% endblock header %}#}
-
+{# fmt:off #}
```

Comment:
@helenb This is typically added just before breaking into Jinja code (and removed right after). We still want formatter rules to apply to block content in general, so should probably stick with that convention

Reply:
As mentioned on slack it kept doing strange indenting *outside* the `{#fmt#}` tags when I tried using them closer in to the affected code.